src/fastfeedparser/main.py

```diff
@@ -44,640 +47,573 @@ def parse(source):
-                            "type": link.get("type"),
-                            "href": href,
-                            "title": link.get("title"),
-                        }
-                    )
-
-        # Check for guid that looks like a URL
-        guid = item.find("guid")
-        guid_text = guid.text.strip() if guid is not None and guid.text else None
-        is_guid_url = guid_text and (
-            guid_text.startswith("http://") or guid_text.startswith("https://")
-        )
-
-        if is_guid_url:
-            # Prefer guid as link when it looks like a URL
-            entry["link"] = guid_text
-            if alternate_link:
-                entry["links"].insert(
-                    0, {"rel": "alternate", "type": "text/html", "href": guid_text}
-                )
-        elif alternate_link:
-            entry["links"].insert(0, alternate_link)
-            entry["link"] = alternate_link["href"]
-        elif (
-            "link" not in entry
-            and guid is not None
-            and guid.get("isPermaLink") == "true"
-        ):
-            entry["link"] = guid.text
-
-        content = None
-        if feed_type == "rss":
-            content = item.find("{http://purl.org/rss/1.0/modules/content/}encoded")
-            if content is None:
-                for ns, uri in namespaces.items():
-                    if uri == "http://purl.org/rss/1.0/modules/content/":
-                        content = item.find(f"{{{uri}}}encoded")
-                        break
-            if content is None:
-                content = item.find("content")
-        elif feed_type == "atom":
-            content = item.find("{http://www.w3.org/2005/Atom}content")
-
-        if content is not None:
-            content_type = content.get("type", "text/html")  # Default to text/html
-            if content_type in ["html", "xhtml"]:
-                # For XHTML content, serialize the entire content
-                content_value = etree.tostring(
-                    content, encoding="unicode", method="xml"
-                )
-            else:
-                content_value = content.text if content.text else ""
-
+    # If content is still empty, try to use description
+    if "content" not in entry:
+        description = item.find("description")
+        if description is not None and description.text:
             entry["content"] = [
                 {
-                    "type": content_type,
-                    "language": content.get(
-                        "{http://www.w3.org/XML/1998/namespace}lang"
-                    ),
-                    "base": content.get("{http://www.w3.org/XML/1998/namespace}base"),
-                    "value": content_value,
-                }
+                    "type": "text/html",
+                    "language": item.get("{http://www.w3.org/XML/1998/namespace}lang"),
+                    "base": item.get("{http://www.w3.org/XML/1998/namespace}base"),
```

Comment:
For consistency, I add language and base but idk whether they should come from item element or description element.